web/pandas/about/team.md

```diff
@@ -36,10 +36,19 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+
+## Diversity and Inclusion
+    
```

Comment:
There are whitespaces in some blank lines, or at the end of the ones with text. I'd recommend to set up your editor so it shows them, or warns you about them, the continuous integration of pandas checks that they don't exist.